### PR TITLE
Make stateless components into functional components #1681

### LINKED ISSUE #1681: Make stateless components into functional components
**Actual Behaviour**

Following Components are stateless:

- FourButtons
- CircleImage
- Footer
- Logout
- DialogSection
- ThreadListItem
- Devices

**Expected Behaviour**

Convert above components to functional components.

**Would you like to work on the issue?**

Yes

### PR DESCRIPTION
**Pull request by fragm3**
Fixes #1681 
Changes: 
Made read only let to const.

Following Components are made functional components:

- FourButtons
- CircleImage
- Footer
- Logout
- DialogSection
- ThreadListItem
- Devices

Demo Link: https://pr-1681-fossasia-susi-web-chat.surge.sh

### CODE REVIEW

**Comment by fragm3** [comment]
Please ignore codacy.

File: src/components/CircleImage/CircleImage.js [hunk 1]
```diff
@@ -25,93 +25,93 @@ function sumChars(str) {
...
+    colors,
+    style,
+    borderRadius = '100%',
+    className,
+  } = props;
 
-  static defaultProps = {
-    colors: defaultColors,
-  };
+  let size = addPx(props.size);
```
Comment (AakashMallik) [inline src/components/CircleImage/CircleImage.js]: const

File: src/components/CircleImage/CircleImage.js [hunk 2]
```diff
@@ -25,93 +25,93 @@ function sumChars(str) {
...
-      src,
-      srcset,
-      name,
-      color,
-      colors,
-      style,
-      borderRadius = '100%',
-      className,
-    } = this.props;
+  let styles = {
```
Comment (AakashMallik) [inline src/components/CircleImage/CircleImage.js]: const

File: src/components/CircleImage/CircleImage.js [hunk 3]
```diff
@@ -25,93 +25,93 @@ function sumChars(str) {
...
+      marginRight: 10,
+      borderRadius,
+    },
+  };
 
-    let size = addPx(this.props.size);
+  let { imageStyle, innerStyle } = styles;
+  const nameInitials = initials(name);
+  let innerElement,
+    classes = [className, 'CircleImage'];
```
Comment (AakashMallik) [inline src/components/CircleImage/CircleImage.js]: classes could be declared as const here

File: src/components/Team/FourButtons.react.js [hunk 4]
```diff
@@ -1,72 +1,71 @@
...
+const flatButtonStyleNone = {
+  minWidth: '10%',
+  fontSize: '25px',
+  color: '#fff',
+  padding: '0%',
+  display: 'none',
+};
+
+const FourButtons = props => {
+  let member = props.member;
```
Comment (AakashMallik) [inline src/components/Team/FourButtons.react.js]: const

**Comment by fragm3** [comment]
@AakashMallik this PR is concentrated towards making stateless components to functional components.

**Comment by AakashMallik** [comment]
> @AakashMallik this PR is concentrated towards making stateless components to functional components.

I get it. But they're little corrections might not need another PR. Just a suggestion. :innocent:

File: src/components/CircleImage/CircleImage.js (hunk 3 above)
Comment (fragm3) [reply src/components/CircleImage/CircleImage.js]: They are being reassigned, so let it be.

Comment (AakashMallik) [reply src/components/CircleImage/CircleImage.js]: Oh right. Completely missed that. Sorry.

**Pull request merged**